### PR TITLE
Future3 fix venv usage

### PR DESCRIPTION
**Pull request by AlvinSchiller**
- fix installed packages in venv
- move venv in projekt root
- fix node setup
- update comments / messages
- #2112 fix 'not tagged as plugin callable'

### CODE REVIEW

File: src/jukebox/jukebox/multitimer.py [hunk 1]
```diff
@@ -249,6 +249,7 @@ def start(self, iterations=None, wait_seconds_per_iteration=None):
         self._function = self._callee(*self.class_args, iterations=self._iterations, **self.class_kwargs)
         super().start(wait_seconds_per_iteration)
 
+    @plugin.tag
```
Comment (pabera) [inline src/jukebox/jukebox/multitimer.py:252]: Why?

Comment (AlvinSchiller) [reply src/jukebox/jukebox/multitimer.py:252]: https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2112 
Hasn't to do anything with venv, but was a noop fix.

File: installation/routines/setup_jukebox_core.sh [hunk 2]
```diff
@@ -125,8 +124,11 @@ _jukebox_core_install_settings() {
 
 _jukebox_core_register_as_service() {
   echo "  Register Jukebox Core user service"
-  sudo cp -f "${INSTALLATION_PATH}/resources/default-services/jukebox-daemon.service" "${SYSTEMD_USR_PATH}"
-  sudo chmod 644 "${SYSTEMD_USR_PATH}/jukebox-daemon.service"
+
+  local jukebox_service="${SYSTEMD_USR_PATH}/jukebox-daemon.service"
+  sudo cp -f "${INSTALLATION_PATH}/resources/default-services/jukebox-daemon.service" "${jukebox_service}"
+  sudo sed -i "s|%%INSTALLATION_PATH%%|${INSTALLATION_PATH}|g" "${jukebox_service}"
```
[the remark this answers is not in the document]
Comment (AlvinSchiller) [reply installation/routines/setup_jukebox_core.sh:130]: yep, that will be on the roadmap ;)

File: requirements.txt [hunk 3]
```diff
@@ -13,10 +19,9 @@ requests
 # For the publisher event reactor loop:
 tornado
 
-# RPi's GPIO packages: these are installed via APT on the PI
-# On regular machines, install them manually if needed for development
-# RPi.GPIO
-# gpiozero
+# RPi's GPIO packages:
+RPi.GPIO
```
[the remark this answers is not in the document]
Comment (AlvinSchiller) [reply requirements.txt:23]: But they have been installed before via apt ( python3-rpi.gpio python3-gpiozero ) and that worked also in docker as far as i know, if the raspberry sources are added. V2 also uses rpi.gpio with pip

**Review by pabera — APPROVED** [review]

**Pull request merged**